src/main/java/edu/usf/cutr/gtfsrtvalidator/api/resource/GtfsRtFeed.java

```diff
@@ -152,8 +169,13 @@ public Response startMonitor(
         session.save(sessionModel);
         GTFSDB.commitAndCloseSession(session);
 
-        //Extract the Url and gtfsId to start the background process
-        startBackgroundTask(gtfsRtFeed, updateInterval);
+        if(feedSession == null) {
```

Comment:
@mohangandhiGH Yes, I agree - I think there are a few potential issues here.  I've commented in https://github.com/CUTR-at-USF/gtfs-realtime-validator/pull/231#discussion_r123358139 that I don't think this is necessary to prevent a duplicate `BackgroundTask` for the same GTFS and GTFS-rt feed, as we already have some logic in `startBackgroundTask()` for this (see above link for details - I welcome feedback though).